docs/config_to_support_additional_workspace_directories.md

```diff
@@ -0,0 +1,101 @@
+## Config to support additional workspace directories
+
+### Issue
+https://github.com/runfinch/finch/issues/107
+
+### Approach
+
+Lima yaml file has Mounts field to handle the mount points. Similar to other Finch config like cpu and memory, it is straightforward to write additional_workspace_directories to Lima’s override.yaml file.
+
+For example, for Finch config:
+```
+cpus: 2
+memory: 4GiB
+additional_workspace_directories:
+- /Volumes/mydir
+```
+
+Finch config applier will make override.yaml like this:
+
+```
+images: []
+cpus: 2
+memory: 4GiB
+mounts:
+- location: /Volumes/mydir
+mountPoint: /Volumes/mydir
+writable: true
```

Comment:
Option to make it read-only?

Reply:
It is out of scope of this feature and we have not identified the use case of read-only option so far. We can evaluate it based on community's input later.